lib/LoaderRunner.js

```diff
@@ -368,7 +369,7 @@ exports.runLoaders = function runLoaders(options, callback) {
 	iteratePitchingLoaders(processOptions, loaderContext, function(err, result) {
 		if(err) {
 			return callback(err, {
-				cacheable: requestCacheable,
+				cacheable: err instanceof LoaderLoadingError ? false : requestCacheable,
```

Comment:
Instead of doing this, set cacheable to false when `loadLoader` returns an error here:

https://github.com/webpack/loader-runner/blob/ed672e43c8e52e86533c8826c39720f4124f1211/lib/LoaderRunner.js#L171

Reply:
Ok, but just to be clear... Since this can also include the `MODULE_NOT_FOUND` error (which is **not** a `LoaderLoadingError`) - this may introduce a **[breaking change](https://github.com/hulkish/loader-runner/blob/prefer-callback-errors/test/runLoaders.js#L431)**. Current behavior (master) for these errors in master is `cacheable: true`

I actually agree that this is probably a good change to mark not cacheable since they are essentially fatal errors. Just wanted to be clear that this may need a major bump.